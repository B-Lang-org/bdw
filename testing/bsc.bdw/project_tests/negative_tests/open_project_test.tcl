
# Opens the specified project
handlers::open_project 

# Opens the specified project
handlers::open_project project1.bspec

# Opens the specified project
handlers::open_project negative_tests.exp

# Opens the specified project
handlers::new_project proj

# Opens the specified project
handlers::open_project project.bspec

# Closes the project
handlers::close_project

# Exits the workstation
exit
