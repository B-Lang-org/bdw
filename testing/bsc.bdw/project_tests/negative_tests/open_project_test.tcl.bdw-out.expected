The project name is not specified.
The file project1.bspec doesn't exist.
The negative_tests.exp is not a project file.
The current project has not been saved.
